packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbDefault.stories.tsx

```diff
@@ -65,40 +46,9 @@ function renderButton(item: Item) {
 }
 
 export const Default = () => {
-  const [size, setSize] = React.useState('medium' as BreadcrumbProps['size']);
-  const [breadcrumbType, setBreadcrumbType] = React.useState('item');
   return (
     <>
-      <h2>Playground</h2>
-      <div style={{ display: 'flex', flexGrow: 2 }}>
-        <div>
-          <Label>Size</Label>
-          <RadioGroup
-            aria-labelledby="breadcrumb-size"
-            value={size}
-            onChange={(_, data) => setSize(data.value as BreadcrumbProps['size'])}
-          >
-            <Radio value="small" label="Small" />
-            <Radio value="medium" label="Medium" />
-            <Radio value="large" label="Large" />
-          </RadioGroup>
-        </div>
-        <div>
-          <Label>Breadcrumb type</Label>
-          <RadioGroup
-            aria-labelledby="bredcrumb-type"
-            value={breadcrumbType}
-            onChange={(_, data) => setBreadcrumbType(data.value)}
-          >
-            <Radio value="item" label="Non-interactive item" />
-            <Radio value="button" label="Button" />
-          </RadioGroup>
-        </div>
-      </div>
-      <Breadcrumb aria-label="Small Breadcrumb" size={size}>
-        {breadcrumbType === 'item' && items.map(item => renderItem(item))}
-        {breadcrumbType === 'button' && items.map(item => renderButton(item))}
-      </Breadcrumb>
+      <Breadcrumb aria-label="Breadcrumb default example">{items.map(item => renderButton(item))}</Breadcrumb>
```

Comment:
I realized that the guidance states

`The last item shouldn't be interactive.`

However, you're still render a BreadcrumbButton, is that expected? or should the user render BreadcrumbItem in this case?

Reply:
Good catch, I'll fix it